src/BloomExe/web/ServerBase.cs

```diff
@@ -75,44 +75,29 @@ protected ServerBase()
 
 		#region Startuup
 
-		public void StartWithSetupIfNeeded()
+		public void StartListening()
```

Comment:
When the parameter was removed from `protected virtual bool StartWithSetupIfNeeded(out Exception error)` one of them needed to be renamed.